### PR TITLE
Improve logging for Zika normalization and docs

### PR DESCRIPTION
**Pull request by kedhammar**
In some cases, automated decisions in the normalization will result in slightly misguided log messages.

E.g. if the buffer volume is omitted because it's less than the minimum transfer amount, the final concentration will be above target and the sample will be reported as "too concentrated" even if it is not.

Also took the opportunity to improve some documentation in one of the Zika utility functions.

### CODE REVIEW

**Comment by chuan-wang** [comment]
Hi @kedhammar, is it OK to submit a new PR for this when [PR#284](https://github.com/SciLifeLab/scilifelab_epps/pull/284) is merged? Otherwise we will have to fix conflicts.

I have pushed the lab today to approve the verification

**Comment by kedhammar** [comment]
Can I wait for the merge of #284 and then pull changes into this PR?

No rush

**Comment by chuan-wang** [comment]
> @chuan-wang can I wait for the merge of #284 and then pull changes into this PR?
> 
> No rush

👍

**Review by chuan-wang — APPROVED** [review]

**Comment by kedhammar** [comment]
Commit made to fix merge conflicts.

Will test functionality again and then re-submit for review.

**Comment by kedhammar** [comment]
Ready to merge, re-requesting review @chuan-wang

File: scripts/zika_utils.py [hunk 1]
```diff
@@ -72,20 +72,51 @@ def __init__(self, log, log_filename, lims, currentStep):
         sys.stderr.write("ERROR: Check log for more info.")
         sys.exit(2)
 
-def fetch_sample_data(currentStep, to_fetch):
+
+def fetch_output_udfs(currentStep):
```
Comment (chuan-wang) [inline scripts/zika_utils.py:76]: `fetch_output_udfs` and `assert_udfs` are not in use any more. Considering deleting them?

Comment (kedhammar) [reply scripts/zika_utils.py]: [c415a87](https://github.com/SciLifeLab/scilifelab_epps/pull/286/commits/c415a87cb7ded6822d6251780e1a911d4b670714)

**Pull request merged**